### DISCUSSION TITLE
Wrong Plots Count ?

### DISCUSSION THREAD

**Discussion by eksion** [swar/Swar-Chia-Plot-Manager#481] [Q&A]
Hello

I'm using swar on ubuntu
I'm using 3x1tb ssd ( Raid One ) and try to do the plot

But it seems the plot count is not right.. 
I only have 5 parallel job running but it shows i have 10 plots running

![IMG_20210517_205946](https://user-images.githubusercontent.com/25521335/118501635-24a69e80-b753-11eb-8610-1269a050040e.jpg)

Am i missing something?

**Comment by Cminer3207** [comment]
Post this in Issues.

**Reply by wyd431966** [reply]
I read code,i think it's processes.py, get_running_plots function issues.The variable work in function is in chia_processes circle.But,chia_processes  every 1 plot have 2 processes.Hope can help.Wish can fix issues.
